restapi/src/main/java/io/stargate/web/docsapi/exception/XNotFoundException.java

```diff
@@ -0,0 +1,12 @@
+package io.stargate.web.docsapi.exception;
+
+/** Thrown when API can not find a requested resource. */
+public class XNotFoundException extends RuntimeException {
```

Comment:
Why `X`? :thinking: What do you think about `ResourceNotFoundException`?

Reply:
Done.